internal/dockerutil/filewriter.go

```diff
@@ -48,10 +47,10 @@ func (w *FileWriter) WriteFile(ctx context.Context, volumeName, relPath string,
 			Cmd: []string{
 				// Take the uid and gid of the mount path,
 				// and set that as the owner of the new relative path.
-				`chown "$(stat -c '%u:%g' "$1")" "$2"`,
+				`chown -R "$(stat -c '%u:%g' "$1")" "$2"`,
 				"_", // Meaningless arg0 for sh -c with positional args.
 				mountPath,
-				path.Join(mountPath, relPath),
+				mountPath,
```

Comment:
I made this change as the hermes relayer ended up making several files in other places in the home directory and was running into permissions issues. I thought it might be fine if we just set the home directory to be owned by the relayer user. If this is a problem we can figure something else out!